pkg/utils/maps/sync_test.go

```diff
@@ -0,0 +1,336 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package maps
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestSyncMap_Delete(t *testing.T) {
+	type args[K comparable] struct {
+		key K
+	}
+	type testCase[K comparable, V any] struct {
+		name         string
+		m            *SyncMap[K, V]
```

Comment:
This one defines local variables directly inside the test function, and the others do the same